tests/kernel/ibtests.pm

```diff
@@ -6,11 +6,9 @@
 # Package: git-core twopence-shell-client bc iputils python
 # Summary: run InfiniBand test suite hpc-testing
 #
-# Maintainer: Michael Moese <mmoese@suse.de>, Nick Singer <nsinger@suse.de>
+# Maintainer: Michael Moese <mmoese@suse.de>, Nick Singer <nsinger@suse.de>, ybonatakis <ybonatakis@suse.com>
 
-use base 'opensusebasetest';
-use strict;
```

Comment:
Is there a reason to not use strict and warnings anymore?

Reply:
https://docs.mojolicious.org/Mojo/Base#toc says it all. no need with _Mojo::Base_